bill_of_materials.md

```diff
@@ -0,0 +1,27 @@
+This repositoay contains the frame for the MIRTE Light, Basic and Pioneer.
+The releases will include files for lasercutting (DXF), 3D printing (STL),
+and files to generate renders for workshops (STEP). 
+
```

Comment:
Maybe add the current thickness (4mm) somewhere in this file

Reply:
Currently the default is 3mm.